### PR TITLE
publish umd bundle to be used as script tag for expressions library

### PR DESCRIPTION
**Pull request by atishayv**
Fixes 
Allow users to consume this library from a distribution umd package which users can include as script tag in their html file.

## Description
Allow users to consume this library from a distribution umd package which users can include as script tag in their html file.
Something like : 
<script src="https://unpkg.com/browse/adaptive-expressions@4.10.4/dist/adaptive-expressions.js"></script>

## Specific Changes
- Used webpack to bundle the library into umd distribution package.
- Added release command in package json scripts to get it 

## Testing
Tested by including this library as script tag in a HTML page

### CODE REVIEW

**Comment by joshgummersall** [comment]
We've been using browserify in other packages to produce browser friendly bundles. Would that suffice rather than adding another webpack dependency?

File: libraries/adaptive-expressions/package.json [hunk 1]
```diff
@@ -50,13 +51,15 @@
     "build": "tsc",
     "build-docs": "typedoc --theme markdown --entryPoint adaptive-expressions --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\adaptive-expressions .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Expression\" --readme none",
     "clean": "erase /q /s .\\lib",
+    "postbuild": "browserify lib/index.js -s AEL -o dist/adaptive-expressions.js",
```
Comment (joshgummersall) [inline libraries/adaptive-expressions/package.json:54]: Can we just place this inside `lib/browser.js`? Also, is it worth adding `"browser": "lib/browser.js"` to `package.json`?

Comment (atishayv) [reply libraries/adaptive-expressions/package.json]: Placed it in lib/browser.js file. 
I have tried adding browser field in package json, but it throws error that it cant walk the dependecy graph. Not sure why.

Comment (joshgummersall) [reply libraries/adaptive-expressions/package.json]: I ran into that as well, I can resolve once this gets merged.

**Review by joshgummersall — APPROVED** [review]

File: libraries/adaptive-expressions/package.json (hunk 1 above)
Comment (atishayv) [reply libraries/adaptive-expressions/package.json]: Thanks

**Pull request merged**